### ISSUE TITLE
Add pointers or a tutorial for adding tools to the toolbox with pure JS

### ISSUE THREAD

**Issue by cubiclesoft** [GetmeUK/ContentTools#395]
> The code examples in this tutorial use CoffeeScript as this is the recommended language to use to extend the editor, but it is possible to use JavaScript if you prefer.

If I want to add a tool to the toolbox, the only official tutorial shows a long and convoluted process for adding new tools with CoffeeScript.  That's great for adding defaults to the core product but not so great for allowing third-party inclusion to customize the toolbox on the fly later on.  The documentation/tutorial says pure-JS is possible but then doesn't offer any pointers on making it happen.  This isn't a request for plugin support but for the existing tutorial to be updated with pointers on a pure JS approach to accomplish the same task OR add a dedicated JS tutorial for those of us who don't want to/can't use CoffeeScript for whatever reason.

**Comment by anthonyjb** [comment]
@cubiclesoft Agreed - I think wherever a CoffeeScript example is presented a JavaScript example should also be included. I'll make this a rule for new content updates and look to apply it retrospectively to existing articles.
